src/lib/integration_test_cloud_engine/kubernetes_network.ml

```diff
@@ -59,36 +114,43 @@ module Node = struct
     let path = Printf.sprintf "/%s/graphql" node.app_id in
     Uri.make ~scheme:"http" ~host ~path ~port:80 ()
 
-  let get_logs_in_container ?container_id { pod_ids; config; pod_info; _ } =
+  let get_logs_in_container ?container_id t =
     let container_id =
-      Option.value container_id ~default:pod_info.primary_container_id
+      Option.value container_id ~default:t.pod_info.primary_container_id
     in
     let%bind cwd = Unix.getcwd () in
+    let open Malleable_error.Let_syntax in
+    let%bind pod_ids = refresh_pod_ids_of_node_hard_error t in
     Integration_test_lib.Util.run_cmd_or_hard_error ~exit_code:13 cwd "kubectl"
-      ( base_kube_args config
+      ( base_kube_args t.config
       @ [ "logs"; "-c"; container_id; List.hd_exn pod_ids ] )
 
   let run_in_container ?(exit_code = 10) ?container_id ?override_with_pod_id
       ~cmd t =
     let { config; pod_info; _ } = t in
+    let%bind cwd = Unix.getcwd () in
```

Comment:
Why move this?

Reply:
I just wanted to move the boilerplate code to the top, aesthetically thought it was better.  If you have reasons against, I can move it back